src/menu_items.c

```diff
@@ -3565,14 +3773,12 @@ void load_menu_img_mio0_forced(MenuTexture* addr) {
     load_menu_img_comp_type(addr, LOAD_MENU_IMG_MIO0_FORCE);
 }
 
-void load_menu_img_comp_type(MenuTexture* addr, s32 compType) {
+void load_menu_img_comp_type(MenuTexture* texAddr, s32 compType) {
```

Comment:
This function is a regression. Please grab the latest of this function and fix the names.

Reply:
I just simplify the function (and avoid a bunch of useless step because we use lus to manage texture)